OAuthSwiftTests/OAuthSwiftNetworkActivityNotifierTests.swift

```diff
@@ -0,0 +1,310 @@
+//
+//  OAuthSwiftNetworkActivityNotifierTests.swift
+//  OAuthSwiftTests
+//
+//  Created by Jarek Pendowski on 24/07/2019.
+//  Copyright © 2019 Dongri Jin. All rights reserved.
+//
+
+import XCTest
+@testable import OAuthSwift
+
+class OAuthSwiftNetworkActivityNotifierTests: XCTestCase {
+    
+    let url = URL(string: "http://github.com/pendowski/OAuthSwift")!
+
+    func testMockNotRunning() {
+        let timeout = expectation(description: "Timeout")
+        let handler = MockedNetworkHandler()
+        
+        _ = handler.dataOperation(with: URLRequest(url: url)) { (data, response, error) in
+            XCTFail()
+        }
+        
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 0.1) {
```

Comment:
`0.01` may be fine and noticeably faster when you sum up time spent in all tests.